website/content/docs/updates/important-changes.mdx

```diff
@@ -187,30 +188,22 @@ roles behave as expected.
 
 ### Recommendation
 
-If you rely on LDAP static roles, upgrade to Vault 1.19.2+, 1.18.8+, 1.17.15+,
-or 1.16.19+.
+If you rely on LDAP static roles, upgrade to Vault 1.19.3+, 1.18.9+, 1.17.16+,
+or 1.16.20+.
```

Comment:
I changed this from 1.19.2 -> 1.19.3. I dont think we should recommend anyone upgrade to 1.19.0-1.19.2 if they can avoid it